src/components/Form.js

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
 
 export default function Form() {
+  const dispatch = useDispatch();
+  const [author, setAuthor] = useState('');
+  const [title, setTitle] = useState('');
+
+  const submitForm = (e) => {
+    e.preventDefault();
+    dispatch(addBook({ title, author, id: Date.now() }));
```

Comment:
- [ ] I was unable to remove a book I added. You need to use `item_id`, not `id` here.